OCCommunicationLib/OCCommunicationLib/OCCommunication.h

```diff
@@ -599,11 +600,44 @@ typedef enum {
              successRequest:(void(^)(NSHTTPURLResponse *response, NSArray *listOfShared, NSString *redirectedServer)) successRequest
              failureRequest:(void(^)(NSHTTPURLResponse *response, NSError *error, NSString *redirectedServer)) failureRequest;
 
+
+///-----------------------------------
+/// @name shareFileOrFolderByServerPath:withFileOrFolderPath:password:expirationTime:publicUpload:linkName:onCommunication
+///-----------------------------------
+
+/**
+ * Method to share a file or folder with password
+ *
+ * @param serverPath -> NSString server path
```

Comment:
Better explanations is needed; ```server path``` could mean just the path in the URL, after domain. This parameter corresponds to the full base URL to the server.